### ISSUE TITLE
Plot Loss vs Epoch Curve as in Paper

### ISSUE THREAD

**Issue by Deepanshu-beep** [kenshohara/3D-ResNets-PyTorch#230]
@kenshohara As you plotted a curve for Validation loss vs Epoch in your Paper.
How can I plot the same curve?
